rclpy/CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ endfunction()
 
 add_library(
   rclpy
-  SHARED src/rclpy/_rclpy.c
+  src/rclpy/_rclpy.c
```

Comment:
This can never be static though right? Otherwise Python would not be able to dynamically load it.

Reply:
That was my uncertainty, I wanted to test it to confirm before placing this in review.